traits/01-web-access.Rmd

```diff
@@ -1,4 +1,4 @@
----
+6---
```

Comment:
did you intend to put this six here?

Or should it still be 

```suggestion
---
```

Reply:
No I did not intend to place the 6 there -- I will remove it.